tests/prague/eip7692_eof_v1/eip7069_extcall/test_calls.py

```diff
@@ -811,7 +825,8 @@ def test_eof_calls_no_balance(
             Op.SSTORE(slot_call_result, Op.EXTCALL(address=noop_callee_address, value=1))
```

Comment:
```suggestion
            Op.SSTORE(slot_call_result, Op.EXTCALL(address=noop_callee_address, value=balance + 1))
```
I suggest we instead use `balance + 1` here to test:
- Zero balance, non-zero value transfer
- Non-zero balance, not enough value transfer

And the transfer fails in both cases, instead of testing valid value transfer, which I think should already be covered elsewhere.

Reply:
I think they aren't, successful EXTCALL with value was the case I wanted to add here. Or have I missed them somewhere?

That being said, adding a balance=1 value=2 case makes a lot of sense. I have generalized the test a bit, and I've also updated it name to reflect this properly, PTAL.